README.md

```diff
@@ -148,6 +148,20 @@ Also backlist the audio kernel:
 
 add the `blacklist snd_bcm2835` to the end of the file.
 
+### Hardware PWM
+
+Hardware PWM is required in order to drive higher frequency signals, such as the 25000 Hz needed by Noctua PWM fans. The rpi-hardware-pwm module is used as a lightweight alternative to pigpio. Install the required library: `sudo pip install rpi-hardware-pwm`

```

Comment:
you can avoid manually installing by adding `rpi-hardware-pwm` to the setup_requires list [here](https://github.com/shortbloke/OctoPrint-Enclosure/blob/63dc9919b0c14d71a0261a02a560899b72881e48/setup.py#L36)